css/_settings_toolbar.scss

```diff
@@ -0,0 +1 @@
+/** Insert custom CSS for any additional content in the welcome page settings toolbar **/
```

Comment:
From the name of the file it is not clear that the file is related to the welcome page settings toolbar. Could you please add 'welcome_page' in the name?